packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -108,8 +108,12 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
         opacity: !_showTitle ? 1.0 : 0.0,
         duration: const Duration(milliseconds: 250),
         child: FloatingActionButton(
+          backgroundColor: Theme.of(context).colorScheme.surface,
```

Comment:
There must be a default for something as important as a `FloatingActionButton`. My first guess is `primary`, but perhaps `secondary`.